util/src/main/java/io/grpc/util/ForwardingLoadBalancer.java

```diff
@@ -30,8 +30,8 @@ public abstract class ForwardingLoadBalancer extends LoadBalancer {
   protected abstract LoadBalancer delegate();
 
   @Override
-  public void handleResolvedAddresses(ResolvedAddresses resolvedAddresses) {
```

Comment:
This will need some more thought. This change would prevent extending classes from seeing the addresses, if they are only overriding handleResolvedAddresses().

Reply:
Can we mark this existing method as deprecated and introduce an override for acceptResolvedAddresses, please confirm

Reply:
That would still break an implementation only implementing `handleResolvedAddresses()`